base/changes.txt

```diff
@@ -6,6 +6,11 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2023-08-19 Joseph Wright  <Joseph.Wright@latex-project.org>
+
+	* ltcmd.dtx, usrguide.tex
+	Remove \GetDocumentCommandArgSpec, etc.
```

Comment:
Guessing Frank may ask to list them all.

Reply:
Done

Reply:
> Guessing Frank may ask to list them all.

:-) but yes, if I had noticed